rpc/common/types.go

```diff
@@ -37,6 +37,17 @@ type C struct {
 	NotConnected    int `json:"not-connected"`
 }
 
+// ConsensusInternal captures consensus internal data
+type ConsensusInternal struct {
+	ViewID        uint64 `json:"viewid"`
+	VCID          uint64 `json:"vcid"`
+	Mode          string `json:"mode"`
+	Phase         string `json:"phase"`
+	BlockNum      uint64 `json:"blocknum"`
+	InSync        bool   `json:"insync"`
+	ConsensusTime int64  `json:"finality"`
```

Comment:
`vcid` is not clear IMO. Maybe also stick to [kebab case](https://wiki.c2.com/?KebabCase) for the unmarshalled names if we decide to keep it in node metadata. Or use snake case.